src/test/java/uk/gov/hmcts/reform/laubackend/cases/service/AccessRequestServiceTest.java

```diff
@@ -112,49 +109,72 @@ void shouldThrowExceptionWhenTimestampIsUnparsable() {
+        Page<AccessRequest> accessRequestPage = new PageImpl<>(
+            accessRequestList,
+            PageRequest.of(0, 10, Sort.by("timestamp").descending()),
+            1
         );
 
-        verify(accessRequestFindRepository, times(1)).findAll(accessRequest,null, pageRequest);
-        List<AccessRequestLog> accessRequestLogs = accessRequestGetResponse.getAccessLog();
-        assertThat(accessRequestLogs).hasSize(1);
-        AccessRequestLog accessLog = accessRequestLogs.getFirst();
+        when(accessRequestFindRepository
+                 .findAll(queryParams, null,
+                          PageRequest.of(0, 10, Sort.by("timestamp").descending())
+                 ))
+            .thenReturn(accessRequestPage);
+
+        // When
+        AccessRequestGetResponse response = accessRequestService.getAccessRequestRecords(queryParams);
+
+        // Then
+        verify(accessRequestFindRepository, times(1))
+            .findAll(queryParams, null,
+                     PageRequest.of(0, 10, Sort.by("timestamp").descending())
+            );
+
+        assertThat(response).isNotNull();
+        assertThat(response.getAccessLog()).hasSize(1);
+        assertThat(response.getAccessLog().get(0)).isInstanceOf(AccessRequestLog.class);
+        assertThat(response.getStartRecordNumber()).isEqualTo(1);
+        assertThat(response.isMoreRecords()).isFalse();
+        assertThat(response.getTotalNumberOfRecords()).isEqualTo(1);
+
+
+        AccessRequestLog accessLog = response.getAccessLog().getFirst();
         assertThat(accessLog.getRequestType()).isEqualTo(AccessRequestType.CHALLENGED);
         assertThat(accessLog.getUserId()).isEqualTo("user-id");
         assertThat(accessLog.getCaseRef()).isEqualTo("case-ref");
         assertThat(accessLog.getReason()).isEqualTo("reason");
         assertThat(accessLog.getAction()).isEqualTo(AccessRequestAction.APPROVED);
-        assertThat(accessLog.getRequestStart()).isEqualTo("2021-08-01T00:00:00.000Z");
         assertThat(accessLog.getRequestEnd()).isEqualTo("2021-08-01T23:59:59.999Z");
         assertThat(accessLog.getTimestamp()).isEqualTo("2021-08-01T00:00:00.000Z");
     }
 
     private AccessRequest getAccessRequest() {
-        AccessRequest accessRequest =  new AccessRequest();
+        AccessRequest accessRequest = new AccessRequest();
         accessRequest.setRequestType("CHALLENGED");
         accessRequest.setUserId("user-id");
         accessRequest.setCaseRef("case-ref");
         accessRequest.setReason("reason");
         accessRequest.setAction("APPROVED");
-        accessRequest.setRequestStart(Timestamp.valueOf("2021-08-01 00:00:00.000"));
         accessRequest.setRequestEnd(Timestamp.valueOf("2021-08-01 23:59:59.999"));
         accessRequest.setTimestamp(Timestamp.valueOf("2021-08-01 00:00:00.000"));
         return accessRequest;
     }
+
+    private AccessRequestGetRequest getAccessRequestGetAccessRequest() {
```

Comment:
To me this would be very confusing - almost like a double copy paste "getAccessRequest" and again "getAccessRequest".. Shouldn't this be "getAccessRequestGetRequest" ?